ui/app/scripts/stream/services/metamodel.js

```diff
@@ -176,7 +176,7 @@ define(function (require) {
                                             properties[p.id] = p;
                                         }
                                     }
-                                    if (p.sourceType === 'org.springframework.cloud.stream.app.transform.ProgrammableRxJavaProcessorProperties') {
+                                    if (p.sourceType === 'org.springframework.cloud.stream.app.transform.ProgrammableReactiveProcessorProperties') {
```

Comment:
@BoykoAlex Where is this specific configuration properties class located? I don't seem to find it in spring-cloud-stream-app-starters.